src/Apps/Order/Routes/Shipping2/Components/SaveAndContinueButton.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button, ButtonProps } from "@artsy/palette"
+import { useSaveSelectedShippingQuote } from "Apps/Order/Routes/Shipping2/Hooks/useSaveSelectedShippingQuote"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
+import { SaveAndContinueButton_order$key } from "__generated__/SaveAndContinueButton_order.graphql"
+import { graphql, useFragment } from "react-relay"
+
+interface SaveAndContinueButtonProps {
+  order: SaveAndContinueButton_order$key
+  width?: ButtonProps["width"]
+}
+
+export const SaveAndContinueButton: React.FC<SaveAndContinueButtonProps> = ({
```

Comment:
Noticed most of the behavior in the `Routes/Shipping2/index` file came down to this submit button, so moved code out of there and colocated it directly in a new thing, which can be shared between desktop / mobile views (only that that changed between them was the width of the button)